src/message/MessageList.js

```diff
@@ -40,8 +56,21 @@ export default class MessageList extends React.PureComponent {
   };
 
   render() {
-    const { caughtUp, fetching, fetchOlder, fetchNewer, singleFetchProgress } = this.props;
-    const messageList = renderMessages(this.props);
+    const {
+      caughtUp, fetching, fetchOlder, fetchNewer, singleFetchProgress, auth,
+      doNarrow, subscriptions, messages, narrow, mute
+    } = this.props;
+
+    const args = {
+      'auth': auth,
+      'subscriptions': subscriptions,
+      'messages': messages,
+      'narrow': narrow,
+      'mute': mute,
+      'doNarrow': doNarrow,
+      'handleLongPress': this.openMessageActionSheet
+    };
+    const messageList = renderMessages(args);
```

Comment:
On the other hand, it will not be needed, because I will suggest you handle the long press in `Chat.js`.